ImperatorToCK3/Source/CK3/Province/CK3Province.h

```diff
@@ -47,7 +47,7 @@ class Province
 	  void setReligion(const mappers::ReligionMapper& religionMapper);
 	  void setCulture(const mappers::CultureMapper& cultureMapper);
 	
-	int provID = 0;
+	  unsigned long long provID = 0;
```

Comment:
I do not believe provinces are ullongs. They are specifically limited by what's in definition.csv and don't even break 10k. Having them as ullongs is of course your prerogative, but from experience only people have such IDs.

Reply:
Yeah, it's mostly to standardize and use only string and ullong types for all IDs.